### DISCUSSION TITLE
How to spread an array of length 1

### DISCUSSION THREAD

**Discussion by qianxi0410** [typst/typst#1428] [Q&A] [answered]
In fact, I have a function that is similar to `#let func(array) `，But the length of the array may be 1, so I can't use `..` syntax, how do I handle this elegantly, and do I have to use `#if` to make lengthy judgements

**Comment by laurmaedje** [comment, marked as answer]
You can also use `..` syntax with an array of length one. Make sure to include the trailing comma when constructing an array of length one, otherwise it is just a parenthesized expression, not an array: `func(..(x,))`.

**Reply by qianxi0410** [reply]
Very detailed instructions, thanks!
